### DISCUSSION TITLE
Feature: Sort cars in Grafana differently

### DISCUSSION THREAD

**Discussion by airmaxx** [teslamate-org/teslamate#1333] [Ideas]
**Describe the bug**

Currently my old car "K.I.T.T." is always shown on top in Grafana dashboards, then my current car "Speedy Gonzales". I have to scroll down to see my current car. This is inconvenient, as I'm more interested in the data of my newest car. Maybe sorting currently is alphabetically?

**Expected behavior**

Manually sort cars, or have newest car always on top

**Comment by jpph** [comment]
Yes, would be nice to be able to sort car. Tryed to change name, but it doesn't sort by name. The easiest would be to sort by name so we can rename car (putting _ in front for ex to put it on top)
